core/build.gradle.kts

```diff
@@ -39,15 +39,20 @@ android {
 
     buildFeatures {
         viewBinding = true
+        compose = true
     }
-    namespace = "com.redmadrobot.panel_core"
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = androidx.versions.compose.compiler.get()
+    }
```

Comment:
It looks like extra changes related to migration to compose

Reply:
See https://github.com/RedMadRobot/debug-panel-android/pull/16#discussion_r1580774193